api/src/main/java/org/pmiops/workbench/compliance/ComplianceServiceImpl.java

```diff
@@ -42,6 +61,11 @@ public Integer getMoodleId(String email) throws ApiException {
 
   @Override
   public List<BadgeDetails>  getUserBadge(int userMoodleId) throws ApiException {
+    if (!enableMoodleCalls())
+      return null;
+    setApiHost();
+    api.getApiClient().setDebugging(true);
```

Comment:
Should this perhaps be a config value as we have for Firecloud (`debugEndpoints`)?

Reply:
i meant to remove this but forgot, it was for my testing